daemon/main.go

```diff
@@ -399,6 +400,8 @@ func init() {
 		"nat46-range", node.DefaultNAT46Prefix, "IPv6 prefix to map IPv4 addresses to")
 	flags.BoolVar(&masquerade,
 		"masquerade", true, "Masquerade packets from endpoints leaving the host")
+	flags.IntVar(&option.Config.MTU,
```

Comment:
`flags.IntVar` -> `flags.UIntVar`

Reply:
Do you really care about this? It requires to touch almost every single use of mtu in the code. Do you just care about the user entering negative values? uint is not much better either, the user can still enter invalid values and they will be rejected when we try to apply.

Reply:
> Do you just care about the user entering negative values?

Yes, that was the only reason

Reply:
OK, I'll add a validation check in `pkg/option/config.go`, that is a much simpler fix for this